GSA/usagov-benefits-eligibility
# Add add-to-project action file

## PR Summary
This PR creates a new Github Action to automate `the addition of all the newly created issues to the project board`.

## Related Github Issue

- fixes #[(issue)](https://github.com/orgs/GSA/projects/48/views/10?pane=issue&itemId=24684638)

## Type of change

Please delete options that are not relevant.


## Branch Name

feature/Automate-Github-issues-to-project

## Detailed Testing steps

1. Merge this PR
2. Open a new issue
3. See if it lands on our project board
